doc/src/site/sphinx/migration_guide.rst

```diff
@@ -38,14 +38,19 @@ DRF exposed into Sparkling Water Algorithm API
 
 DRF is now exposed in the Sparkling Water. Please see our documentation to learn how to use it :ref:`drf`.
 
+Change Default Name of Prediction Column
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The default name of the prediction column is changed to ``prediction`` from ``prediction_output``.
```

Comment:
nit: `The default name of the prediction column has been changed from ``prediction_output`` to ``prediction`` `?

Reply:
Yul, like it better!